From 7ef356e12bd3e58695877111e3420c090170effb Mon Sep 17 00:00:00 2001
From: Delphine CC Chiu <Delphine_CC_Chiu@wiwynn.com>
Date: Tue, 12 Mar 2024 16:24:52 +0800
Subject: [PATCH] ARM: dts: aspeed: yosemite4: Enable ipmb device for OCP debug
 card

Add OCP debug card devicetree config

This commit depends on the patch below:
[1] https://lore.kernel.org/lkml/20240312053320.380997-1-Delphine_CC_Chiu@wiwynn.com/

Signed-off-by: Delphine CC Chiu <Delphine_CC_Chiu@wiwynn.com>
---
 .../boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts    | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts
index a8b724049e83..c736e342a427 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts
@@ -1026,6 +1026,15 @@ imux37: i2c@3 {
 &i2c13 {
 	status = "okay";
 	bus-frequency = <400000>;
+	multi-master;
+
+	ipmb@10 {
+		compatible = "ipmb-dev";
+		reg = <(0x10 | I2C_OWN_SLAVE_ADDRESS)>;
+		i2c-protocol;
+	};
+
+
 	gpio@20 {
 		compatible = "nxp,pca9506";
 		reg = <0x20>;
-- 
2.25.1

